app.rb

```diff
@@ -0,0 +1,163 @@
+require_relative './classroom'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    prompt_user
+  end
+
+  # List all books
+  def list_all_books
```

Comment:
- [ ] Kindly fix the following error on List all Books

![image](https://user-images.githubusercontent.com/92215878/211083053-6ca15ac2-5e00-47b5-b201-c659b3a91010.png)